stable/azure-devops-agent/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{ if .Values.secrets.create }}
```

Comment:
shouldn't this be:
```suggestion
{{ if .Values.azpToken }}
```

Reply:
Looking this again, I can't even see the purpose for the secret if it's expected to pull the PAT from the values file (can't see how the PAT will be provided from the value).  
Having a rethink on this and will test/add updates to read PAT from keyvault using CSI.  Will cancel and issue a new PR once tested.

Reply:
sure